Sources/funnet/Core/Lists.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  Lists.swift
+//  FunNet
+//
+//  Created by Elliot Schrock on 6/12/22.
+//
+
+import UIKit
+import LithoOperators
+import Prelude
+
+public func defaultShouldLoadNextPage(_ pageSize: Int = 20, _ pageTrigger: Int = 5, _ numberOfRows: Int, _ current: Int) -> Bool {
+    return numberOfRows - current == pageTrigger && numberOfRows % pageSize == 0
+}
+
+public extension NetworkCall {
```

Comment:
I really like how this works, but I feel like FunNet is supposed to be a UI-independent network layer. I'm also not sure where else this could go, especially since the impetus for this is that LUX was _too_ layered. If we had more cases where we do something similar (searching comes to mind), then maybe it would warrant a separate subspec

Reply:
I've started working on this again, and am planning to add support for async/await, as well as collapsing some of the subspecs since v13 support is really not a question anymore. So this sort of thing will be coming.